CONTRIBUTING.md

```diff
@@ -42,36 +46,55 @@ be able to work on TabPy changes:
     cd TabPy
     ```
 
-To start a local TabPy instance:
+## Setting Up Environment
+
+Before making any code changes run environment setup script. For
+Windows the next command from the repository root folder:
```

Comment:
this sentence doesn't make sense to me...

Reply:
fixed